Sources/Build/Triple.swift

```diff
@@ -105,12 +106,15 @@ public struct Triple {
 
     public static let macOS = try! Triple("x86_64-apple-macosx10.10")
     public static let linux = try! Triple("x86_64-unknown-linux")
+    public static let ppc64leLnx = try! Triple("powerpc64le-unknown-linux")
```

Comment:
Drive-by nit: could you spell this `ppc64leLinux`?

Reply:
@xwu I have committed necessary changes today. Thanks!